src/azure-cli/azure/cli/command_modules/acs/custom.py

```diff
@@ -1744,17 +1756,23 @@ def _aks_browse(
         logger.warning('Proxy running on %s', proxy_url)
 
     timeout = None
-    if os.getenv("PYTEST_CURRENT_TEST", None):
+    test_hook = check_cmd_test_hook_file("test_aks_browse_legacy_hook.json")
+    if test_hook and test_hook == "enabled":
         timeout = 10
```

Comment:
why 10 seconds?
can it be 1 millisecond?

Reply:
I think 1 millisecond is too short for `kube-proxy` to establish a connection. Especially the test case will deliberately give an illegal address, so that it will try to establish a connection twice within the timeout period. In the live test, compared to the waiting time (about 5 minutes) to create a cluster, the 10-second waiting time hardly prolongs the test time.